src/components/Assets/index.js

```diff
@@ -0,0 +1,10 @@
+import back from "./back.png";
+import logo from "./logo.png"

```

Comment:
Procura mantener la consistencia en el estilo de código.
```suggestion
import logo from "./logo.png";
```